python/ray/tune/examples/README.rst

```diff
@@ -1,4 +1,60 @@
 Tune Examples
 =============
 
-Code examples for various schedulers and Tune features.
+.. Keep this in sync with ray/doc/tune-examples.rst
```

Comment:
Could we just link to the readthedocs here? Is there an advantage of having a README that's buried in the repo?

Reply:
Well the examples here are intended to be actively viewed and perused, so this is a natural landing page IMO.